src/partisan_hiscamp_membership_strategy.erl

```diff
@@ -66,11 +66,11 @@ try_distance(#hiScamp{membership=Membership0, distance=Dist}=_State0, Node, Dict
     Dict = get(distance_metrics),
     case dict:find(Node, Dict) == error of
         true ->
-            erlang:error("Cannot attain node distain");
+            erlang:error("Cannot attain node distance");
         false ->
-            {ok, Dist} -> Dist
+            Val = {ok, Value}
     end,
-    Dist = element{2, Dist}.
+    Dist = element{2, Val}.
```

Comment:
Syntax error.  Element is a function, so it needs to be used with paranethesis, and not braces.